cdap-watchdog/src/main/java/co/cask/cdap/logging/logbuffer/LogBufferService.java

```diff
@@ -51,22 +52,25 @@
 import java.util.concurrent.ExecutionException;
 
 /**
- * Service responsible for loading, starting and stopping log buffer pipelines, creating concurrent writer and
+ * Log Buffer Service responsible for:
+ * loading, starting and stopping log buffer pipelines
+ * creating concurrent writer
+ * starting log buffer cleaner service to clean up logs that have been persisted by log buffer pipelines
+ * recovering logs from log buffer
```

Comment:
`recovering logs from log buffer` - add details on why the logs need recovering

Reply:
The details are added in `LogBufferRecoveryService`. Have not added much info in this service since its a controlling service.